webrtc.html

```diff
@@ -3998,6 +3998,19 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             the <a><code>RTCRtpReceiver</code></a> to be stopped.</p>
           </dd>
 
+          <dt>void activateSender()</dt>
+
+          <dd>
+            <p>The <dfn id="dom-rtptransceiver-activatesender"><code>activateSender</code></dfn>
+            method activates the <code>RTCRtpTransceiver.sender</code> such that future calls
+            to <code>createOffer</code> and <code>createAnswer</code> mark the corresponding
+            media description as <code>sendrecv</code> or <code>sendonly</code>, as defined in
+            [[!JSEP]].
+
+            <p>If the sender is not already active and the PeerConnection is in the stable state,
+            this will trigger <code>onnegotiationneeded</code>.
+          </dd>
```

Comment:
Shouldn't this say "this will set the negotiation-needed flag", and leave off mentioning the stable state?
I assume that this sequence:

pc.createOffer() => offer
pc.setLocal(offer)
track.activateSender()
pc.setRemote(answer)  // state is now stable

expects a "negotiationneeded" to be fired after the setRemote() is called.

Also, "trigger onnegotiationneeded" is language we're not using - it's either "set negotiation-needed" or "fire a negotiationneeded event" (I think the former).